src/main/scala/fs2/async/package.scala

```diff
@@ -0,0 +1,39 @@
+package fs2
+
+import fs2.util.Catchable
+
+
+/**
+ * Created by pach on 10/10/15.
+ */
+package object async {
+
+
+  /**
+   * Create a new continuous signal which may be controlled asynchronously.
+   * Note that this would block any resulting processes (discrete, continuous) until any signal value is set.
+   */
+  def signalUnset[F[_]:AsyncExt,A]: Stream[F,mutable.Signal[F,A]] =
```

Comment:
I'd like to just delete this method. Use `Option` + filtering on the consumer side if you want this.